extensions/amp-analytics/0.1/events.js

```diff
@@ -389,6 +425,42 @@ export class VisibilityTracker extends EventTracker {
   }
 
   /**
+   * @param {string|undefined} waitForSpec
+   * @param {string|undefined} selector
+   * @param {?Element} element
+   * @return {?Promise}
+   * @visibleForTesting
+   */
+  getReadyPromise(waitForSpec, selector, element) {
+    if (!selector) {
+      // Default case #1: wait for nothing if selector is not specified.
+      return null;
```

Comment:
This is not quite correct. It's ok for a root to specify, e.g. `waitFor: 'ini-load'`. I think we should instead have something like this:
```
if (!waitForSpec) {
  // Default.
  if (!selector) {
    wautForSpec = NONE;
  } else {
    waitForSpec = INI_LOAD;
  }
}
if (waitForSpec == NONE) {
  return null;
}
```